packages/live-share-canvas/src/canvas/InkingCanvas.ts

```diff
@@ -107,7 +104,7 @@ export abstract class InkingCanvas {
                 width: clientWidth,
                 height: clientHeight,
                 halfWidth: clientWidth * 0.5,
-                halfHeight: clientHeight * 0.5
+                halfHeight: clientHeight * 0.5,
```

Comment:
I think it tied, and i was the deciding vote (Previously my vote was no preference). I went with the default which is "Trailing commas where valid in ES5 (objects, arrays, etc.). No trailing commas in type parameters in TypeScript".

diffs end up being slightly cleaner when you add another line, since you are not editing the line above to add a comma. Is this something you want to discuss again?

Reply:
Re-vote does not allow no-preference, so there will be no ties. If the user declines to vote, that will count towards default value.